common/config/config.go

```diff
@@ -576,13 +591,51 @@ type (
 		// URI is the domain default URI for visibility archiver
 		URI string `yaml:"URI"`
 	}
+
+	// YamlNode is a lazy-unmarshaler, because *yaml.Node only exists in gopkg.in/yaml.v3, not v2,
+	// and go.uber.org/config currently uses only v2.
+	YamlNode struct {
```

Comment:
How does it work?

Reply:
Nevermind, I was reading doc of v3 package instead of v2.

Reply:
yea.  I tried to use v3 first, and then the `Load` tests caught the issue :|

thankfully it's pretty simple to achieve in v2.